sql/src/main/java/org/apache/druid/sql/calcite/planner/ExplainAttributes.java

```diff
@@ -117,6 +120,31 @@ public String getReplaceTimeChunks()
     return replaceTimeChunks;
   }
 
+  @Override
+  public boolean equals(Object o)
+  {
+    if (this == o) {
+      return true;
+    }
+    if (o == null || getClass() != o.getClass()) {
+      return false;
+    }
+    ExplainAttributes that = (ExplainAttributes) o;
+    return Objects.equals(statementType, that.statementType) && Objects.equals(
+        targetDataSource,
+        that.targetDataSource
+    ) && Objects.equals(partitionedBy, that.partitionedBy) && Objects.equals(
+        clusteredBy,
+        that.clusteredBy
+    ) && Objects.equals(replaceTimeChunks, that.replaceTimeChunks);
```

Comment:
Formatting: please put each `Objects.equals()` in a separate line.

Reply:
Ah yeah, not sure why Intellij formatted it weirdly. Fixed.